Marsiple/App/Albums/AlbumsView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AlbumsView.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class AlbumsView: UIView {
+    private var albums = [Album]()
+    let tableView = UITableView()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension AlbumsView: UITableViewDataSource {
```

Comment:
The `View` should `never` contain `business` logic.  The `View` only contains layout and styling code.

The `UITableViewDataSource` conformance should be moved to the `view controller` level (as should the `private var albums = [Album]()` property.)